src/utils/services/mapError.js

```diff
@@ -0,0 +1,28 @@
+const mapError = (state) => {
+  let errorMessage = '';
+  switch (state) {
+    case 'EMAIL_EXISTS':
+      errorMessage = 'The email address is already in use by another account.';
+      break;
```

Comment:
It could be simplified to just return the string. 
Ex.:
```js
const mapError = (state) => {
switch (state) {
case 'EMAIL_EXISTS':
return 'The email address...';
case ...
```